From 6b926b041e939655d3a88202bde32f4901459a65 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Mon, 4 Sep 2023 22:36:10 +0200
Subject: [PATCH] arch: arm64: dts: green: Improve LED representation

Fix color and use labels/node names according to the LEDs functionality.
---
 arch/arm64/boot/dts/rockchip/rk3566-ha-green.dts | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-ha-green.dts b/arch/arm64/boot/dts/rockchip/rk3566-ha-green.dts
index e4b1e80641b9..1d0db11db3e0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-ha-green.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-ha-green.dts
@@ -48,16 +48,18 @@ hdmi_con_in: endpoint {
 	leds {
 		compatible = "gpio-leds";
 
-		led_power: led-0 {
+		led_power: led-power {
+			label = "power";
 			gpios = <&gpio0 RK_PB6 GPIO_ACTIVE_HIGH>;
 			function = LED_FUNCTION_POWER;
-			color = <LED_COLOR_ID_RED>;
+			color = <LED_COLOR_ID_WHITE>;
 			default-state = "keep";
 			linux,default-trigger = "default-on";
 			pinctrl-names = "default";
 			pinctrl-0 = <&led_power_pin>;
 		};
-		led_act: led-1 {
+		led_activity: led-activity {
+			label = "activity";
 			gpios = <&gpio0 RK_PB7 GPIO_ACTIVE_HIGH>;
 			function = LED_FUNCTION_ACTIVITY;
 			color = <LED_COLOR_ID_GREEN>;
@@ -65,7 +67,8 @@ led_act: led-1 {
 			pinctrl-names = "default";
 			pinctrl-0 = <&led_act_pin>;
 		};
-		led_user: led-2 {
+		led_user: led-user {
+			label = "user";
 			gpios = <&gpio0 RK_PC5 GPIO_ACTIVE_HIGH>;
 			function = LED_FUNCTION_HEARTBEAT;
 			color = <LED_COLOR_ID_YELLOW>;
